src/Our.Umbraco.Vorto/Web/PropertyEditors/VortoPropertyEditor.cs

```diff
@@ -73,7 +73,10 @@ internal class VortoPreValueEditor : PreValueEditor
 
             [PreValueField("hideLabel", "Hide Label", "boolean", Description = "Hide the Umbraco property title and description, making the Vorto span the entire page width")]
             public bool HideLabel { get; set; }
-		}
+
+            [PreValueField("showFilledLanguages", "Show Languages With Content", "boolean", Description = "Show a tick-mark next to the language name when there is content for that language.")]
+            public bool ShowFilledLanguages { get; set; }
```

Comment:
I'm not sure if "show" is the right word as it suggests something would be hidden. Maybe something like "Mark languages with content" or something?